modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullscreenRestoreScrollTest.ts

```diff
@@ -0,0 +1,52 @@
+import { Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Css, Insert, InsertAll, Remove, Scroll, SugarBody, SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/fullscreen/Plugin';
+
+describe('browser.tinymce.plugins.fullscreen.FullscreenRestoreScrollTest.ts', () => {
```

Comment:
I don't think there should be a `ts` extension here.

Reply:
🤦  that was my bad sorry